mod-director-core/src/main/java/net/jan/moddirector/core/configuration/type/UrlRemoteMod.java

```diff
@@ -25,18 +25,24 @@
     private final String fileName;
     private final URL url;
     private final String[] follows;
+    private final String folderName;
+    private String inject;
```

Comment:
Instead of adding these here, add them into `ModDirectorRemoteMod`, as they apply to all mod types.